draft-deletecookie-weiss-http.md

```diff
@@ -77,8 +77,34 @@ This document uses the following terminology from Section 3 of {{STRUCTURED-FIEL
 The Delete-Cookie response header is a Structured Field {{STRUCTURED-FIELDS}} List of strings.
 Each string represents a cookie {{COOKIES}} name to be deleted.
 
-A user agent that receives a Delete-Cookie response header MUST delete cookies for the server's registrable domain with corresponding names from its cookie store.
-Each name could match multiple cookies of that same name, set on different paths.
+A user agent that receives a Delete-Cookie response header MUST process it before any Set-Cookie headers received as part of the same response.
+
+*Note:* The relevant processing needs to happen before step 15 of the [network fetch](https://fetch.spec.whatwg.org/#http-network-fetch) algorithm.
+
+When receiving a Delete-Cookie response header, the user agent MUST:
+
+1. If the response was not delivered over a secure channel {{COOKIES}}, return.
+1. Let `host` be the canonicalized response URL's host name.
+1. Let `parsed` be the result of parsing the header's value as a List, as per {{STRUCTURED-FIELDS}}, Section 4.2.1.
+1. If parsing fails, return.
+1. For each `name` in `parsed`:
+   1. If `name` is not a String {{STRUCTURED-FIELDS}}, continue.
+
+        *Note:* The above ignores any parameters in the header's value List.
```

Comment:
Nit: I feel like the spacing of all the notes in this section is somewhat inconsistent. Have you checked the output?

Reply:
Yeah. They *seem* fine..